docs/demo/gaiden-css/components/gallery-item/demo.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+  <link rel="stylesheet" href="/gaiden-css/gaiden.css">
+  <link rel="stylesheet" href="/gaiden-css/base.css">
+</head>
+<body>
+  <section class="container">
+    <a class="col-small-6 col-medium-6 col-normal-4 col-large-3 space-box-medium" href="#link">
+      <dl class="gallery-item">
```

Comment:
Nice using the article tag, but I would not use the picture tag and header tag. The two tags are used to represents a group.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header